packages/web-pkg/src/services/permissionManager.ts

```diff
@@ -1,6 +1,21 @@
 import { Store } from 'vuex'
+interface RoleSetting {
```

Comment:
```suggestion
interface Permission {
```
technically, backend side, it's a setting of the role. But semantically, in the context of a role, it's a permission.